### ISSUE TITLE
Delete v1alpha1 operator

### ISSUE THREAD

**Issue by alculquicondor** [kubeflow/mpi-operator#422]
And migrate samples to the v2 controller. Notably `examples/mxnet`

/help

**Comment by Jeffwan** [comment]
I briefly check all the dependencies. I think the major problem Aldo encountered was master used too old kubernetes and old volcano dependencies. 

```
volcano.sh/apis v1.2.0-k8s1.16.15
k8s.io/api => k8s.io/api v0.15.10
k8s.io/apimachinery => k8s.io/apimachinery v0.15.10
k8s.io/apiserver => k8s.io/apiserver v0.15.10
k8s.io/client-go => k8s.io/client-go v0.15.10
k8s.io/code-generator => k8s.io/code-generator v0.15.10
```

and v2 uses 1.19 which is aligned with existing training operator setting.  We've actually handles these issues in the past. See follow issues for details

https://github.com/kubeflow/common/pull/87 
https://github.com/kubeflow/common/pull/123
https://github.com/kubeflow/common/pull/127
https://github.com/kubeflow/common/pull/126


In conclusion, I think there's no technical blocking issue to put v1 and v2beta1 in same go module. while v1 code needs some updates to support `context` etc (required after 1.19) and this should be done anyway. 

That means we can move forward with https://github.com/kubeflow/training-operator/pull/1413 and then migrate v2 if some one has capacity.  WDYK?  @alculquicondor @zw0610  @kubeflow/wg-training-leads

**Comment by zw0610** [comment]
Sounds good. Since our holiday week just ended, let me check with @hackerboy01 when we can remove `WIP`.

**Comment by terrytangyuan** [comment]
Sounds good to me.

**Comment by zw0610** [comment]
> I think the priority could be the other way around, given that v2 already has up-to-date dependencies: Move v2 first and then v1 if there is extra capacity.

From the pr to merge v1 controller https://github.com/kubeflow/training-operator/pull/1413/files, I found the there is no dependency changes. So technically, we won't need priority for merging the two versions: they can be merged independently.

**Comment by alculquicondor** [comment]
Can the same contributor work on the v2 migration?
